packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -4333,6 +4342,21 @@ export default class Meeting extends StatelessWebexPlugin {
       data: {trigger: trigger.USER_INTERACTION, isRoapCallEnabled: true},
     });
 
+    if (!isEmpty(this.meetingInfo)) {
+      Metrics.postEvent({
+        event: eventType.MEETING_INFO_REQUEST,
+        meeting: this,
+      });
+
+      Metrics.postEvent({
```

Comment:
so we're sending the metric for request and response together at the time of join and not when we actually fetch meeting info? (I just want to double check that CA is OK with that)

Reply:
Yes, I've confirmed with Meera before doing this, and we only want to send the meeting info events once at the beginning of the meeting, but we're doing fetch meeting info several times and in several different contexts, hence we can't send them all the time.

This was just one solution Arun and I came up with and CA team said it works for them too.